src/content/api.yml

```diff
@@ -3665,17 +3665,17 @@ body:
 
   - p: >
       The following condition is also automatically included when the [platform](#platform)
```

Comment:
I think that especially the change to mention `neutral` platform here is especially important because otherwise this could be misread by other implementers.